datarobot_provider/example_dags/advanced_datarobot_pipeline_jdbc_dag.py

```diff
@@ -0,0 +1,229 @@
+        task_id="get_jdbc_credentials",
+        credentials_param_name="datarobot_jdbc_connection",
+    )
+
+    create_project_op = CreateProjectOperator(
+        task_id='create_project',
+        dataset_id=dataset_connect_op.output,
+        credential_id=get_jdbc_credentials_op.output,
+    )
+
+    train_models_op = TrainModelsOperator(
+        task_id="train_models",
+        project_id=create_project_op.output,
+    )
+
+    autopilot_complete_sensor = AutopilotCompleteSensor(
+        task_id="check_autopilot_complete",
+        project_id=create_project_op.output,
+    )
+
+    deploy_model_op = DeployRecommendedModelOperator(
+        task_id="deploy_recommended_model",
+        project_id=create_project_op.output,
+    )
+
+    update_segment_analysis_settings_op = UpdateSegmentAnalysisSettingsOperator(
+        task_id="update_segment_analysis_settings",
+        deployment_id=deploy_model_op.output,
+    )
+
+    update_bias_and_fairness_settings_op = UpdateBiasAndFairnessSettingsOperator(
+        task_id="update_bias_and_fairness_settings",
+        deployment_id=deploy_model_op.output,
+    )
+
+    update_monitoring_settings_op = UpdateMonitoringSettingsOperator(
+        task_id="update_monitoring_settings",
+        deployment_id=deploy_model_op.output,
+    )
+
+    get_jdbc_connection_op = GetOrCreateDataStoreOperator(
+        task_id="get_jdbc_connection",
+        connection_param_name="datarobot_jdbc_connection",
+    )
+
+    score_predictions_op = ScorePredictionsOperator(
+        task_id="score_predictions_jdbc",
+        deployment_id=deploy_model_op.output,
+        intake_datastore_id=get_jdbc_connection_op.output,
+        output_datastore_id=get_jdbc_connection_op.output,
+        intake_credential_id=get_jdbc_credentials_op.output,
+        output_credential_id=get_jdbc_credentials_op.output,
+    )
+
+    scoring_complete_sensor = ScoringCompleteSensor(
+        task_id="check_scoring_complete",
+        job_id=score_predictions_op.output,
+    )
+
+    service_stats_op = GetServiceStatsOperator(
+        task_id="get_service_stats",
+        deployment_id=deploy_model_op.output,
+    )
+
+    target_drift_op = GetTargetDriftOperator(
+        task_id="target_drift",
+        deployment_id=deploy_model_op.output,
+    )
+
+    feature_drift_op = GetFeatureDriftOperator(
+        task_id="feature_drift",
+        deployment_id=deploy_model_op.output,
+    )
+
+    @task(task_id="example_processing_python")
+    def service_stat_processing(model_service_stat, feature_drift, target_drift, accuracy):
+        """Example of custom logic based on service stats from the deployment."""
+
+        # Put your service stat processing logic here:
+        current_model_id = model_service_stat['model_id']
+        total_predictions = model_service_stat['metrics']['totalPredictions']
+        print(f"model_id:{current_model_id}, total_predictions:{total_predictions}")
```

Comment:
How are prints displayed in Airflow? Is it better to print or to log from a DAG?